bigframes/_config/bigquery_options.py

```diff
@@ -235,6 +268,10 @@ def kms_key_name(self) -> Optional[str]:
         Cloud KMS CryptoKey Encrypter/Decrypter IAM role in the key's project.
         For more information, see https://cloud.google.com/bigquery/docs/customer-managed-encryption#assign_role
         Assign the Encrypter/Decrypter.
+
+        Returns:
+            None or str:
+                Customer managed encryption key as a string; otherwise None.
```

Comment:
It's not the actual key, it's the name of the key.

Reply:
Done.